process/dataindexer/dataIndexer.go

```diff
@@ -77,21 +75,62 @@ func (di *dataIndexer) SaveBlock(outportBlock *outport.OutportBlock) error {
 		return err
 	}
 
-	wi := workItems.NewItemBlock(
-		di.elasticProcessor,
-		&outport.OutportBlockWithHeader{
-			OutportBlock: outportBlock,
-			Header:       header,
-		},
-	)
-	di.dispatcher.Add(wi)
+	headerHash := outportBlock.BlockData.HeaderHash
+	shardID := header.GetShardID()
+	headerNonce := header.GetNonce()
+	defer func(startTime time.Time, headerHash []byte, headerNonce uint64, shardID uint32) {
```

Comment:
we can remove all parameters from this list in order to simplify the call. We need the `startTime := time.Now` variable.

Reply:
done